akka-http-core/src/test/scala/akka/http/impl/engine/client/HttpsProxyGraphStageSpec.scala

```diff
@@ -54,7 +54,7 @@ class HttpsProxyGraphStageSpec extends AkkaSpecWithMaterializer {
         connectMsg.utf8String should startWith(
           new EnhancedString(s"""CONNECT $targetHostName:$targetPort HTTP/1.1
             |Host: $targetHostName
-            |Proxy-Authorization
+            |Proxy-Authorization: Basic dGVzdFVzZXJuYW1lOnRlc3RQYXNzd29yZA==
```

Comment:
Oops, embarassing we missed that while reviewing 866fbf8356ea7bdf3025f42d8e87b251fe103f08